tests/check/check_signed_video_auth.c

```diff
@@ -491,9 +491,15 @@ START_TEST(remove_one_p_nalu)
   // authenticity is NOT OK.
   signed_video_accumulated_validation_t final_validation = {
       SV_AUTH_RESULT_NOT_OK, false, 14, 13, 1, SV_PUBKEY_VALIDATION_NOT_FEASIBLE, true, 0, 0};
+  // SIPPSIPPSIPPSI
+  //
+  // SI                ->   (valid) -> .P
+  //  IPPSI            ->   (valid) -> .....P
+  //      IPPSI        -> (invalid) -> NNNNP
```

Comment:
I think it actually adds a missing item after the first I-frame, but I'm not sure. Hence, it could be that it becomes "NMNNNP".